.openshift-ci/slack/notify.sh

```diff
@@ -1,6 +1,15 @@
 #!/usr/bin/env bash
 set -eo pipefail
 
+# Notify only on the main branch
+if [[ -z "$BRANCH" ]]; then
+    export BRANCH="$(echo "$JOB_SPEC" | jq -r '.extra_refs[0].base_ref')"
+fi
+
+if [[ ! "$BRANCH" =~ ^(master|main)$ ]]; then
```

Comment:
IDK if we have this in this repo, but there is a `is_in_PR_context` helper in both `stackrox/stackrox` and `stackrox/collector`, we might want to reuse that.
https://github.com/stackrox/collector/blob/03a2c942ae835d8c441ded7b29d19e71ed875d4e/.openshift-ci/drivers/scripts/lib.sh#L63

Reply:
Yep, but it's not included here yet, it's going to be another follow-up step.

Reply:
Here is the follow up: https://github.com/stackrox/kernel-packer/pull/231